### PR TITLE
mgr-bootstrap script #1822

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/19323
https://github.com/uyuni-project/uyuni-docs/issues/1822

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
See section 4.1:
[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/9876554/suse_manager_client-configuration_guide.pdf)

**Comment by keichwa** [comment]
@agraul gh proposed you as a reviewer, and that's why I picked you.  Feel free to tag some other developer, if you do not have time or what else ;)

File: modules/client-configuration/pages/registration-bootstrap.adoc [hunk 1]
```diff
@@ -38,10 +38,31 @@ If you register openSUSE Leap 15 or SLE 15 systems using Python 2, the bootst
...
+
+The arguments [option]``--activation-keys`` and [option]``--script``, are the only mandatory arguments.
+On the {productname} Server, as root at the command line execute it with the mandatory arguments.
+Replace [systemitem]``<ACTIVATION_KEYS`` and [systemitem]``<EDITED_NAME>`` with your values:
+
+----
+mgr-bootstrap --activation-key=<ACTIVATION_KEYS> --script=bootstrap-<EDITED NAME>.sh
+----
+
+The [command]``mgr-bootstrap`` command offers several other options, including the ability to set a specific hostname, set specific GPG keys, and set the registration method (traditional, salt-minion, salt-bundle).
```
Comment (0rnela) [inline modules/client-configuration/pages/registration-bootstrap.adoc:54]:
```suggestion
The [command]``mgr-bootstrap`` command offers several other options, including the ability to set a specific hostname, set specific GPG keys, and set the registration method (traditional, salt-minion, or salt-bundle).
```

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/registration-bootstrap.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/registration-bootstrap.adoc:54]: thanks!

**Pull request merged**